packages/core/examples/nonVolumeActorAPI/index.ts

```diff
@@ -0,0 +1,225 @@
+import {
+  RenderingEngine,
+  Types,
+  Enums,
+  getRenderingEngine,
+  CONSTANTS,
+} from '@cornerstonejs/core';
+import {
+  initDemo,
+  setTitleAndDescription,
+  addButtonToToolbar,
+  addDropdownToToolbar,
+  camera as cameraHelpers,
+} from '../../../../utils/demo/helpers';
+
+import vtkActor from '@kitware/vtk.js/Rendering/Core/Actor';
+import vtkSphereSource from '@kitware/vtk.js/Filters/Sources/SphereSource';
+import vtkMapper from '@kitware/vtk.js/Rendering/Core/Mapper';
+
+// This is for debugging purposes
+console.warn(
+  'Click on index.ts to open source code for this example --------->'
+);
+
+const { ViewportType } = Enums;
+const { ORIENTATION } = CONSTANTS;
+
+const renderingEngineId = 'myRenderingEngine';
+const viewportId = 'POLYDATA_SAGITTAL';
+
+// ======== Set up page ======== //
+setTitleAndDescription(
```

Comment:
we need to add this example in the `example-info.json` so that it appears in the website